.azure-pipelines/templates/test-all-checks.yml

```diff
@@ -381,8 +381,11 @@ jobs:
       displayName: System Swap
       os: linux
     - checkName: tcp_check
-      displayName: TCP
+      displayName: TCP (Linux)
       os: linux
+    - checkName: tcp_check
+      displayName: TCP (Windows)
```

Comment:
Is this due to a specific issue on windows?

Reply:
For a similar reason we added Windows testing to the `dns_check`, time gets handled slightly differently on the two platforms.